app/controllers/stats_controller.rb

```diff
@@ -27,13 +27,8 @@ def index
     # checked against the whitelist of valid options
     sort_options = ""
     @sort = ""
-    if current_user.preference.hide_hit_counts
-      sort_options = %w(date kudos.count comments.count bookmarks.count subscriptions.count word_count)
-      @sort = sort_options.include?(params[:sort_column]) ? params[:sort_column] : "kudos.count"
-    else
-      sort_options = %w(hits date kudos.count comments.count bookmarks.count subscriptions.count word_count)
-      @sort = sort_options.include?(params[:sort_column]) ? params[:sort_column] : "hits"
-    end
+    sort_options = %w(hits date kudos.count comments.count bookmarks.count subscriptions.count word_count)
```

Comment:
The note above this says "NOTE: Because we are going to be eval'ing the @sort variable later we MUST make sure that its content is checked against the whitelist of valid options" -- does that mean it needs to be done this way, or can it be reduced to just these two lines?